src/main.py

```diff
@@ -6,45 +6,13 @@
 import sys
 
 # Import MEG runtime
-from meg_runtime import Config, PluginManager, ui_run
+from meg_runtime import App, Config, PluginManager
 from meg_runtime.logger import Logger
```

Comment:
Probably only need `from import meg_runtime import App`, I don't think the rest are needed directly there.